Mathlib/LinearAlgebra/Basis/Flag.lean

```diff
@@ -79,6 +80,15 @@ theorem flag_le_ker_coord (b : Basis (Fin n) R M) {k : Fin (n + 1)} {l : Fin n}
   nontriviality R
   exact b.flag_le_ker_coord_iff.2 h
 
+theorem flag_le_ker_dual (b : Basis (Fin n) R M) (k : Fin n) :
+    b.flag k ≤ LinearMap.ker (b.dualBasis k) := by
```

Comment:
```suggestion
    b.flag k.castSucc ≤ LinearMap.ker (b.dualBasis k) := by
```
The other one is not in simp-normal form (and not following the style of other lemmas in this file).